### ISSUE TITLE
Tow Only with Tow Rope in Inventory

### ISSUE THREAD

**Issue by HellBz** [sethduda/AdvancedTowing#36]
is Possible to  tow only if a Player has a Rope in the Inventory ??

This will be able more realism

**Comment by damonsmithies** [comment]
I agree, I have been on servers where this is required and would love for this to be a feature. The ease of this mod to install beats all of the others so don't want to switch to what ever they were using!

**Comment by HellBz** [comment]
Any Updates ?

**Comment by vault-thirteen** [comment]
Well, it looks like this repository received updates 8 years ago, so I consider it to be abandoned. The Arma III game itself is also not really updated by its developers (B.I.S.) while they are working on the next game in the series.
